app/Resources/translations/messages.de.xliff

```diff
@@ -21,7 +21,7 @@
 
             <trans-unit id="title.homepage">
                 <source>title.homepage</source>
-                <target><![CDATA[Willkommen zur <strong>Symfony Demo Application</strong>]]></target>
+                <target><![CDATA[Willkommen zur <strong>Symfony Demo</strong> applikation]]></target>
```

Comment:
Thanks. I've changed it by `Anwendung` because it's the only one that looks like real German :)